themes/src/main/resources/theme/keycloak/welcome/theme.properties

```diff
@@ -1,7 +1,7 @@
-styles=css/welcome.css
 import=common/keycloak
 
-stylesCommon=node_modules/patternfly/dist/css/patternfly.css node_modules/patternfly/dist/css/patternfly-additions.css
+stylesCommon=node_modules/@patternfly-v5/patternfly/patternfly.min.css node_modules/@patternfly-v5/patternfly/patternfly-addons.css
+styles=css/welcome.css
 
-documentationUrl=https://www.keycloak.org/documentation.html
-displayCommunityLinks=true
```

Comment:
I've removed these properties, which might be a breaking change for themes extending the default welcome theme.